03-sre/01-dev-test/tekton-config/clustertasks/stakater-app-sync-and-wait-v1.yaml

```diff
@@ -59,9 +59,26 @@ spec:
           secretKeyRef:
             name: github-stakater-tekton-bot
             key: password
+      - name: ARGOCD_SERVER
+        valueFrom:
+          secretKeyRef:
+            name: argocd-cluster
+            key: server
+      - name: ARGOCD_USERNAME
+        valueFrom:
+          secretKeyRef:
+            name: argocd-cluster
+            key: username
+      - name: ARGOCD_PASSWORD
+        valueFrom:
+          secretKeyRef:
+            name: argocd-cluster
+            key: password
       workingDir: $(workspaces.source.path)
       script: |
         if [ $(params.prNumber) != "NA" ] && [ ! -f pom.xml ]; then
+          argocd login "$ARGOCD_SERVER" --username="$ARGOCD_USERNAME" --password="$ARGOCD_PASSWORD"
+          argocd app get "$(params.argoPreviewAppPrefix)-$(params.repoName)" --refresh
           while [[ "$SECONDS" -lt $(params.timeout) ]]
```

Comment:
you are using timeout param but removing it in param spec

Reply:
timeout is pre defined arg with 60s value, initially i increased it because app wasnt refreshing or syncing with github, but now with line 81, app is refreshed imediately and synced